mlflow/models/evaluation/default_evaluator.py

```diff
@@ -0,0 +1,330 @@
+import mlflow
+from mlflow.models.evaluation.base import (
+    ModelEvaluator,
+    EvaluationMetrics,
+    EvaluationArtifact,
+    EvaluationResult,
+)
+from mlflow.entities.metric import Metric
+from mlflow.utils.file_utils import TempDir
+from mlflow.tracking.artifact_utils import get_artifact_uri
+from sklearn import metrics as sk_metrics
+import matplotlib.pyplot as pyplot
+import scikitplot
```

Comment:
scikitplot is not installed by default in MLR (Databricks Machine Learning Runtime). Can we use different utility(s) for plotting that are available by default on Databricks?